src/main/java/frc/robot/commands/wrist/WristToAngle.java

```diff
@@ -0,0 +1,46 @@
+package frc.robot.commands.wrist;
+
+import edu.wpi.first.wpilibj.command.InstantCommand;
+import frc.robot.Robot;
+import frc.robot.RobotMap;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/******************************************************************************************************************************** 
+** ELEVATOR JOYSTICK COMMAND
+********************************************************************************************************************************/
+public class WristToAngle extends InstantCommand {
+
+  private double mTargetAngle;
+  private final Logger mLogger = LoggerFactory.getLogger(WristToAngle.class);
+
+  /****************************************************************************************************************************** 
+  ** CONSTRUCTOR
+  ******************************************************************************************************************************/
+  public WristToAngle(double targetAngle) {
+    mTargetAngle = targetAngle;
+    requires(Robot.mWrist);
+  }
+
+  /****************************************************************************************************************************** 
+  ** COMMAND OVERRIDES
+  ******************************************************************************************************************************/
+  @Override
+  protected void initialize() {
+    // mLogger.info("Starting WristToAngle command: [{}]", mTargetAngle);
+    // // setInterruptible(true);
+    // Robot.mWrist.MotionMagicOutput(Robot.mWrist.degreesToSensorTicks(mTargetAngle) * RobotMap.kWristGearing + 100);
+    Robot.mWrist.MotionMagicOutput(3300 + RobotMap.kWristTickGain);
```

Comment:
Looks like we still need to generate an equation

Reply:
Line 33 is the theoretic equation, but it produces a much less stable wrist than the manual declaration

Reply:
If (3300 + RobotMap.kWristTickGain) is the same as (Robot.mWrist.degreesToSensorTicks(mTargetAngle) * RobotMap.kWristGearing + 100) for holding the wrist horizontal, then there's no reason the stability will be different
Did you verify these were the same?